flink-kubernetes/src/test/java/org/apache/flink/kubernetes/KubernetesResourceManagerDriverTest.java

```diff
@@ -364,16 +361,16 @@ protected void preparePreviousAttemptWorkers() {
 
         @Override
         protected void validateInitialization() throws Exception {
-            assertNotNull(getPodCallbackHandler());
+            assertThat(getPodCallbackHandler()).isNotNull();
         }
 
         @Override
         protected void validateWorkersRecoveredFromPreviousAttempt(
                 Collection<KubernetesWorkerNode> workers) {
-            assertThat(workers.size(), is(1));
+            assertThat(workers.size()).isEqualTo(1);
```

Comment:
```suggestion
            assertThat(workers).hasSize(1);
```
Please also check for other similar instances.